bin/migrate-mongo.js

```diff
@@ -106,14 +110,18 @@ program
     global.options = options;
     migrateMongo.database
       .connect()
-      .then(({db, client}) => migrateMongo.status(db, client))
-      .then(statusItems => printStatusTable(statusItems))
-      .then(() => {
-        process.exit(0);
+      .then(({db}) => {
```

Comment:
Im not sure we want to do this as nested promises as this breaks the db connection error handler :thinking:
Maybe something like this would work?
```js
  .action(async options => {
    global.options = options;
    try {
      const { db } = await migrateMongo.database.connect();
      const config = await migrateMongo.config.read();

      const statusItems = await migrateMongo.status(db, config.saveFileContents);

      printStatusTable(statusItems);
      process.exit(0);
    } catch (error) {
      handleError(err);
    }
  })
```

Reply:
I had to make `printStatusTable` async as well but otherwise this works, changed